gr-digital/grc/variable_header_format_default.block.yml

```diff
@@ -15,12 +15,17 @@ parameters:
     label: Payload Bits per Symbol
     dtype: int
     default: 1
-value: ${ digital.header_format_default(access_code, threshold, bps) }
+-   id: value
+    label: Resulting access code
+    dtype: string
+    default: 'digital.header_format_default(access_code, threshold, bps)'
+    hide: 'all'
+value: ${ value }
 
 templates:
     imports: from gnuradio import digital
     var_make: |-
-        % if int(eval(access_code))==0:
+        % if access_code == '0':
```

Comment:
What happens if one puts `0000`? Should this result in an all zero access code? Or should this also trigger the default access code? At least this does change the blocks behavior.